debugger/vscode-dap-extension/README.md

```diff
@@ -0,0 +1,51 @@
+# StarlingMonkey Debugger for VS Code
+
+This extension adds support for debugging JavaScript code running in the [StarlingMonkey](https://github.com/bytecodealliance/StarlingMonkey/) JS runtime inside a [WebAssembly Component](https://component-model.bytecodealliance.org/).
+
+**NOTE:** This extension is experimental for now, and will break in surprising and not always enjoyable ways.
+
+## Using The StarlingMonkey Debugger
+
+The extension works for content built with a recent (`0.18` and up) version of [ComponentizeJS](https://www.npmjs.com/package/@bytecodealliance/componentize-js), provided some requirements are met.
+
+### Building Content
+
+**SAMPLE WORKSPACE? JUST RUN THIS:**
+
+**`js` workspace:** `componentize-js --wit world.wit -o main.wasm --runtime-args "--enable-script-debugging" main.js`
+
+**`ts` workspace:** `npm run compile`
+
+**`spin-ts` workspace:** `spin build`
+
+(you will need to `npm install -g @bytecodealliance/componentize-js`)
+
+**MOAR DEETS**
+
+The content needs to be invoked via the `wasi:http/incoming-handler@0.2.*/handle` function. In other words, it needs to target the `wasi:http/proxy@0.2` world.
```

Comment:
I'd hope that with the recent changes to `componentize-js` and StarlingMonkey this is no longer needed:
- https://github.com/bytecodealliance/StarlingMonkey/pull/249
- https://github.com/bytecodealliance/ComponentizeJS/pull/237